pillar/top.sls

```diff
@@ -4,6 +4,12 @@ base:
     - common
     - environment_settings
     - fluentd
+  'P@environment:(rc.*|.*-qa|ci)':
```

Comment:
We don't have any `CI` environments in any of the AWS infrastructure. That's a Heroku only concept.

Reply:
I added it because I saw this:
```
proxy-mitxpro-ci:
    ci
```
Shall I skip that?

Reply:
Yeah, the proxy minions don't actually have any host that they interact with and only exist for populating settings on Heroku.

Reply:
OK. I've removed "ci" in commit 80439006.